src/Mod/Draft/DraftGui.py

```diff
@@ -131,7 +131,13 @@ def __init__(self,widget,extra=None):
         else:
             self.form = widget
     def getStandardButtons(self):
-        return int(QtWidgets.QDialogButtonBox.Close)
+        try:
+            # pyside2
+            return int(QtWidgets.QDialogButtonBox.Close)
+        except TypeError:
+            # pyside6
+            return QtWidgets.QDialogButtonBox.Close.value
+
```

Comment:
this seems like a separate problem and I would like to have this addressed separately to avoid conflicts with other PRs, see https://github.com/FreeCAD/FreeCAD/pull/13611/files#diff-fdf0c359ca3d9da6b4c4614e349806528f209f38894ad323977bc78da2f1ccbe

Reply:
Thanks, but I meant to leave this as it is in main before this PR as it is also being touched by another PR that is pending review

Reply:
took care of it, will merge once CI passes